cluster/addons/metadata-proxy/gce/metadata-proxy.yaml

```diff
@@ -27,26 +27,34 @@ spec:
       dnsPolicy: Default
       containers:
       - name: metadata-proxy
-        image: gcr.io/google-containers/metadata-proxy:0.1.3
-        imagePullPolicy: Always
+        image: gcr.io/google_containers/metadata-proxy:v0.1.4
+        # TODO(#8867) Remove privileged securityContext once iptables munging
+        # is in ip-masq-agent.
         securityContext:
           privileged: true
-        command:
-          - '/start-proxy.sh'
         resources:
           requests:
             memory: "32Mi"
             cpu: "50m"
           limits:
             memory: "32Mi"
             cpu: "50m"
+      # BEGIN_PROMETHEUS_TO_SD
+      - name: prometheus-to-sd-exporter
+        image: gcr.io/google_containers/prometheus-to-sd:v0.2.2
+        command:
+          - /monitor
+          - --stackdriver-prefix={{ prometheus_to_sd_prefix }}/addons/metadata_proxy
+          - --api-override={{ prometheus_to_sd_endpoint }}
+          - --source=metadata_proxy:http://127.0.0.1:989?whitelisted=request_count
         volumeMounts:
-          - name: config-volume
-            mountPath: /etc/nginx/
+        - name: ssl-certs
+          mountPath: /etc/ssl/certs
+      # END_PROMETHEUS_TO_SD
       nodeSelector:
         beta.kubernetes.io/metadata-proxy-ready: "true"
       terminationGracePeriodSeconds: 30
       volumes:
-      - name: config-volume
-        configMap:
-          name: metadata-proxy-config
+      - name: ssl-certs
```

Comment:
Please remove this volume, this is not a requirement for prom-to-sd and will be removed across all of K8s soon

Reply:
Yay!  Done.